packages/ember-runtime/lib/system/array_proxy.js

```diff
@@ -33,6 +35,40 @@ var EMPTY = [];
 
 function K() { return this; }
 
+export function removeAt(array, start, len) {
+  if ('number' === typeof start) {
```

Comment:
why extract this? it doesn't let you fire the events for this operation without using Ember.A().